docker-compose.yml

```diff
@@ -10,7 +15,5 @@ services:
     volumes:
       - tofnd:/.tofnd
     environment:
-      # ATTENTION: use --unsafe only for testing
-      - UNSAFE=true
-      # available cmds: create (default), stored, update, import, export
-      - MNEMONIC_CMD=create 
+      # available cmds: auto, create, existing (default), import, export
```

Comment:
auto is the default here

Reply:
I made the same comment in #194 and it was fixed there then unfixed by the rebase 😛

Reply:
I don't understand what happened... I am sure I had these changed commited at #194

Reply:
oh I see. If you refer to [this](https://github.com/axelarnetwork/tofnd/pull/194#discussion_r718583329) comment, these changes are committed fine.